eng/common/scripts/common.ps1

```diff
@@ -24,6 +24,9 @@ $PackageRepository = "Unknown"
 $packagePattern = "Unknown"
 $MetadataUri = "Unknown"
 
+# Whether the language repo supports automatically loading .env file generated from TestResources scripts.
+$SupportsDotenv = $false
```

Comment:
Perhaps we name this SupportsDotenvForTestResourceOutputs or something to make it clear what it is intended for.

Reply:
The comment directly above it didn't do that? Longer variables just invites the increase risk of typos. I don't see a clear advantage, especially when we have variables like `$packagePattern` just above that. No idea based on the variable name what that does.

Reply:
The comment there helps but it doesn't help in the usages. I agree I don't like long names, but I do like self describing names, especially for global variables like this.

Reply:
Alright, I'll think of something. But I hope this same applies for other vars going forward because several of those above it aren't self-explanatory either...and have no comments describing them, unlike mine.

Reply:
Went with `$SupportsTestResourcesDotenv`.